android/libraries/rib-base/src/main/java/com/badoo/ribs/core/Node.kt

```diff
@@ -321,10 +322,13 @@ open class Node<V : RibView>(
 
     @CallSuper
     open fun handleBackPress(): Boolean {
-//        ribRefWatcher.logBreadcrumb("BACKPRESS", null, null)
-        return plugins.filterIsInstance<BackPressHandler>().any { it.handleBackPressBeforeDownstream() }
+        val subtreeHandlers = plugins.filterIsInstance<SubtreeBackPressHandler>()
+        val handlers = plugins.filterIsInstance<BackPressHandler>()
+
+        return subtreeHandlers.any { it.handleBackPressFirst() }
```

Comment:
preHandleBackPress? is it possible to find out some better naming?)

Reply:
It's not pre- though. It's just the first one we're asking when it happens. Unsure how to express it better with another name.